docker-compose.yml

```diff
@@ -22,7 +22,7 @@ services:
         depends_on:
             - sciencebeam
     sciencebeam:
-        image: elifesciences/sciencebeam:${SCIENCEBEAM_COMMIT}
+        image: elifesciences/sciencebeam:${SCIENCEBEAM_TAG}
```

Comment:
also https://github.com/elifesciences/sciencebeam-texture/pull/28/files#diff-4e5e90c6228fd48698d074241c2ba760R17 needs to change. The tag of the `elifesciences/sciencebeam` image is passed to the build process of the `elifesciences/sciencebeam_texture` image, where it is stored as the label. The formula takes out this label on deploy to choose which sciencebeam to use; the formula itself then seems not to need changes as long as the value is still a valid Docker image tag.

Reply:
Okay, that makes sense now.